pkg/odo/cli/list/list.go

```diff
@@ -258,7 +258,10 @@ func humanReadableOutput(list api.ResourcesList) {
 
 		// If we are managing that component, output it as blue (our logo colour) to indicate it's used by odo
 		if managedBy == "odo" {
-			managedBy = text.Colors{text.FgBlue}.Sprint("odo")
+			managedBy = text.Colors{text.FgBlue}.Sprintf("odo (%s)", comp.ManagedByVersion)
```

Comment:
I am late to the review party ^^, but just wanted to comment that `comp.ManagedByVersion` could be empty if the component is not running yet in any modes, which causes the `odo list` output to be a bit weird to me (`odo ()`), e.g.:

```
❯ odo list         
 ✓  Listing components from namespace 'default' [235ms]
 NAME           PROJECT TYPE  RUNNING IN  MANAGED 
 * my-comp    go                          None               odo ()  
```

Reply:
Ah, better late than never! I'll fix it.

Reply:
:) Not sure what should be displayed, so I created this issue: https://github.com/redhat-developer/odo/issues/6061